src/Umbraco.Cms.Integrations.Automation.Zapier/App_Plugins/UmbracoCms.Integrations/Automation/Zapier/dashboard.html

```diff
@@ -22,50 +23,33 @@
                     You can initiate your automation when a content item of a particular document type is published in Umbraco.
```

Comment:
It would be nice if we could update this to say "... or a form is submitted on your website" if the forms integration is also installed.  I.e. if `vm.formsExtensionInstalled` is `true`.

Reply:
Updated.